gson/src/main/java/com/google/gson/internal/bind/util/ISO8601Utils.java

```diff
@@ -164,8 +164,8 @@ public static Date parse(String date, ParsePosition pos) throws ParseException {
       int hour = 0;
       int minutes = 0;
       int seconds = 0;
-      int milliseconds =
-          0; // always use 0 otherwise returned date will include millis of current time
+      // always use 0 otherwise returned date will include millis of current time
```

Comment:
I'm never sure whether a standalone `//` comment is supposed to apply to the previous line or the next one. Perhaps put a blank line before it to clarify?